x/gamm/keeper/multihop_test.go

```diff
@@ -11,21 +11,33 @@ import (
 	poolincentivestypes "github.com/osmosis-labs/osmosis/v12/x/pool-incentives/types"
 )
 
+var (
+	fooCoin   = sdk.NewCoin("foo", sdk.NewInt(1000000000))
+	barCoin   = sdk.NewCoin("bar", sdk.NewInt(1000000000))
+	bazCoin   = sdk.NewCoin("baz", sdk.NewInt(1000000000))
+	uosmoCoin = sdk.NewCoin("uosmo", sdk.NewInt(1000000000))
+)
+
 func (suite *KeeperTestSuite) TestBalancerPoolSimpleMultihopSwapExactAmountIn() {
 	type param struct {
-		routes            []types.SwapAmountInRoute
-		tokenIn           sdk.Coin
-		tokenOutMinAmount sdk.Int
+		routes             []types.SwapAmountInRoute
+		incentivizedGauges []uint64
```

Comment:
just curious, whats the significance of having incentivized Gauges for testing multihop?

Reply:
Part of the new spec is reduced fees don't happen to ALL osmo-x pool pairs, but instead only incentivized osmo-x pool pairs! (this prevents potential misuse)